spark/dl/src/main/scala/com/intel/analytics/bigdl/example/int8/Quantize.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.example.int8
+
+import com.intel.analytics.bigdl.dataset.image.CropCenter
+import com.intel.analytics.bigdl.dataset.{DataSet, Sample, SampleToMiniBatch}
+import com.intel.analytics.bigdl.nn.Module
+import com.intel.analytics.bigdl.transform.vision.image.augmentation.{ChannelScaledNormalizer, RandomCropper, RandomResize}
+import com.intel.analytics.bigdl.transform.vision.image.{ImageFeature, ImageFrameToSample, MatToTensor, PixelBytesToMat}
+import com.intel.analytics.bigdl.utils.Engine
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.SparkContext
+
+object Quantize {
```

Comment:
I think this example is confused for users. Not clear about how to get scales.

Reply:
Yes. This only generate the scales.